src/flags/index.js

```diff
@@ -20,7 +20,7 @@ export type Flag <T> = {
   completion?: ?{
     cacheDuration?: ?number,
     cacheKey?: ?string,
-    options: () => Promise<string[]>
+    options: (?any) => Promise<string[]>
```

Comment:
I need anything that responds to `.get`, I can't really enforce the client type, can I? We have both `Command`'s and `HerokuCommand`'s, so can't enforce a command type (plus requiring `cli-engine-heroku` here feels like a bad idea)